offchain/indexer/src/main.rs

```diff
@@ -4,14 +4,14 @@
 use clap::Parser;
 
 use indexer::{CLIConfig, IndexerConfig};
-use tracing::info;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let config: IndexerConfig = CLIConfig::parse().into();
 
     log::configure(&config.log_config);
 
-    info!(?config, "Starting Indexer");
+    log::log_service_start(&config, "Starting Indexer");
```

Comment:
```suggestion
    log::log_service_start(&config, "Indexer");
```

To be consistent with the other services.